reactDashboard/src/components/DataDefinitions/DataDefinitions.css

```diff
@@ -58,10 +71,28 @@
   font-weight: bold;
 }
 
+.darkmode .data-definitions .defined-term,
+.darkmode .data-definitions .definitions-heading {
+  color: #c1def1;
+}
+
+.darkmode .data-definitions hr {
+  border: 1px solid #cccccc;
+}
+
 .data-definitions-body .source-attribution {
   color: #2a84bb;
 }
 
-.data-definitions .definition-paragraph {
-  padding-bottom: 20px !important;
+.darkmode .data-definitions .card-body {
```

Comment:
Do you also want to be overriding source-attribution as in the block above?

Reply:
@Boopster

Reply:
I have added an overriding block for darkmode and fixed the contrast error on the wave report this threw when there was no override